src/styles/navstyles.css

```diff
@@ -100,6 +143,37 @@
     height: var(--navbar-scrolled-height)
   }
   
+
+/*For a more user friendly mobile navigatio bar*/
+  nav.responsive_nav {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: auto;
+    height: auto;
+    background-color: var(--color-navyblue);
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+    padding: 2ren 1rem;
+    box-shadow: -2px 0 5px rgba(0, 0, 0, 0.2);
+    transition: all 0.3s ease-in-out;
+    z-index: 10;
+    padding-top: 40px;
+    padding-bottom: 40px;
+  }
+
+  nav.responsive_nav a {
+    text-align: center;
+  }
+
+  .nav-close-btn {
+    align-self: flex-end;
+    font-size: 1.5rem;
+    color: white;
+  }
+
+
   /*.navbar-grid-right {
```

Comment:
Please place any commented out or unused code into the [Knowledge Board](https://docs.google.com/document/d/1BXvILLekHlO0pEJhnBhob7JFHt3supxyvAGyhev482M/edit?usp=sharing) and summarize what it does and why it's been removed. You can then delete this code. Be sure to comment on your new code as well so we understand what the styling is trying to acheive for each class.